### PR TITLE
Add WDI charts

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

**Comment by tjann** [comment]
image
image
image
image
image
image
image
https://screenshot.googleplex.com/isKiTCdLwhAjKGq
image
https://screenshot.googleplex.com/BiPM7PLBL7oajEP
image

File: static/js/shared/stats_var_titles.ts [hunk 1]
```diff
@@ -45,9 +45,23 @@ const STATS_VAR_TITLES: { [key: string]: string } = {
     "Gross National Income (Purchasing Power Parity) Per Capita",
   Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity:
     "Gross National Income (Purchasing Power Parity)",
+  "WorldBank/SL_TLF_0714_ZS": "Children Ages 7-14 in Employment",
```
Comment (tjann) [inline static/js/shared/stats_var_titles.ts:48]: Should I revert these changes, since there is no ranking for int'l vars right now?

**Comment by tjann** [comment]
@shifucun  could you leave this comment "@googlebot I consent."?

**Comment by shifucun** [comment]
@googlebot I signed it!

**Comment by shifucun** [comment]
@googlebot I consent.

File: static/js/shared/stats_var_titles.ts (hunk 1 above)
Comment (beets) [reply static/js/shared/stats_var_titles.ts:48]: please keep it

Comment (tjann) [reply static/js/shared/stats_var_titles.ts:48]: Ack

File: static/js/shared/stats_var_titles.ts [hunk 2]
```diff
@@ -210,15 +231,18 @@ const STATS_VAR_TITLES: { [key: string]: string } = {
...
     "Percentage of People with Physical Health Not Good",
+  "WorldBank/SH_STA_WAST_ZS": "Wasting among Children under 5",
+  "WorldBank/SH_SVR_WAST_ZS": "Severe Wasting among Children under 5",
 
   // Behaviors
   Percent_Person_SleepLessThan7Hours:
     "Percentage of People Who Sleep Less Than 7 Hours",
   Percent_Person_Obesity: "Percentage of People with Obesity",
-  Percent_Person_BingeDrinking: "Percentage of People with Binge Drinking",
+  Percent_Person_BingeDrinking: "Percentage of People That Binge Drink",
```
Comment (beets) [inline static/js/shared/stats_var_titles.ts:241]: along the lines of the fixes, should be lower case "that"

File: static/js/shared/stats_var_titles.ts [hunk 3]
```diff
@@ -210,15 +231,18 @@ const STATS_VAR_TITLES: { [key: string]: string } = {
...
   Percent_Person_PhysicalInactivity:
     "Percentage of People with Physical Inactivity",
-  Percent_Person_Smoking: "Percentage of People who Smoking",
+  Percent_Person_Smoking: "Percentage of People That Smoke",
```
Comment (beets) [inline static/js/shared/stats_var_titles.ts:244]: ditto

**Review by beets — APPROVED** [review]
thanks for all the fixes

File: static/js/shared/stats_var_titles.ts (hunk 2 above)
Comment (tjann) [reply static/js/shared/stats_var_titles.ts:241]: Thanks for pointing out the complexity for title case... I checked all the standards in https://capitalizemytitle.com/ for "that" and they all have it capitalized... maybe we can do sentence case going forward?

**Comment by beets** [comment]
Yea, recommendation from i18n team was also to use sentence case. We should
do it all together to be consistent, maybe outside the scope of this PR

**Comment by tjann** [comment]
I will merge and ignore Codacy which is just saying that our StatVar DCIDs are not camel cased:

image
